webapp/cas/login.uml

```diff
@@ -0,0 +1,29 @@
+@startuml
```

Comment:
Can you write a comment in the file how to view this? If it converts to a .png, add it to a Makefile that creates the png?